### PR TITLE
Add render tests to Gradle project root at `platform/android`

### PR DESCRIPTION
**Pull request by louwers**
And some other Android development tasks

- Update README
- Remove some Mapbox references
- Fix release workflow (it was failing because local.properties was present on CI)
- Lower acceptable delta of one instrumentation test

### CODE REVIEW

**Comment by HarelM** [comment]
Someone else would need to review this as it's specific to Android...

File: platform/android/DEVELOPING.md [hunk 1]
```diff
@@ -33,101 +13,64 @@ git submodule update --init --recursive
...
+Open the `platform/android` directory to get started.
 
-**Note**: We partially support C++14 because GCC 4.9 does not fully implement the
-final draft of the C++14 standard. More information in [DEVELOPING.md](../../DEVELOPING.md).
+## Setting an API Key
 
-**Note**: On macOS you can install clang with installing the [Apple command line developer tools](https://developer.apple.com/download/).
-
-## Opening the project
+_The test application (used for development purposes) uses MapTiler vector tiles, which require a MapTiler account and API key._
```
Comment (cgalvan) [inline platform/android/DEVELOPING.md:24]: Could we have the test application use the MapLibre demotiles instead, so the developer wouldn't need to create a MapTiler account?

https://github.com/maplibre/demotiles/blob/gh-pages/README.md

**Review by cgalvan — APPROVED** [review]

**Comment by louwers** [comment]
@HarelM OK I will keep that in mind!

File: platform/android/DEVELOPING.md (hunk 1 above)
Comment (louwers) [reply platform/android/DEVELOPING.md:24]: It uses the demotiles by default, but many features cannot be demoed with the demotiles style...

**Pull request merged**